src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title: Examples of high-performance CSS animations
+authors:
+  - rachelandrew
+description: |
+  Demonstrating how high performance techniques can create complex and beautiful animations.
+date: 2020-10-20
+tags:
+  - animations
+  - performance
+---
+
+In this post find out how some popular animations found on CodePen have been created.
+THese animations all use the performant techniques discussed in other articles in this section.
+
+See [Why are some animations slow?](/animations-overview/) to learn the theory behind
+these recommendations, and the [Animations Guide](/animations-guide) for practical tips.
+
+## Wizard loading animation
+
+{% IFrame {
+  src: 'https://codepen.io/iamlark/embed/jYzYJg?height=458&theme-id=light&default-tab=result',
+  title: 'Only CSS Loader - Wizard by Guilmain Dorian'
+}
+%}
+
+This loading animation is built entirely with CSS.
+The image plus all of the animation has been created in CSS and HTML,
+no images or JavaScript.
+To understand how it was created and how well it performs you can use Chrome DevTools.
+
+With the animation running, open the Performance tab in Chrome DevTools and record a few seconds of the animation.
+You should see in the Summary that the browser is not doing any Layout or Paint operations when running this animation.
+
+<figure class="w-figure">
+  <img src="./wizard-summary.jpg" alt="Summary in DevTools">
+  <figcaption class="w-figcaption">
+    The summary after profiling the wizard animation.
+  </figcaption>
+</figure>
+
+To find out how this animation was achieved without causing layout and paint,
+inspect any of the moving elements in Chrome DevTools.
+For example select the triangle,
+and watch how the box of the element transforms during its journey into the air,
+as it spins, and then returns to the start position.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/triangle.mp4" type="video/webm">
+  </video>
+  <figcaption class="w-figcaption">
+    Video showing how we can track the path of the triangle in Chrome DevTools.
+  </figcaption>
+</figure>
+
+With the element still selected look at the Styles Panel.
+There you can see the CSS which draws the shape of the triangle,
+and the animation being used.
+
+The triangle is created by using the `::after` pseudo-element to add generated content,
+using borders to create the shape.
+
+```css
+.triangle {
+    position: absolute;
+    bottom: -62px;
+    left: -10px;
+    width: 110px;
+    height: 110px;
+    border-radius: 50%;
+}
+
+.triangle::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: -10px;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 0 28px 48px 28px;
+    border-color: transparent transparent #89beb3 transparent;
+}
+```
+
+{% Glitch 'animation-breakdown1-1' %}
```

Comment:
Thanks for breaking it down but in this case I think we can just provide `animation-breakdown1-2` because it's quite minimal and you've explained all the moving parts (pun legitimately not intended)